### PR TITLE
add yandex API key setting for translate extension

### PR DESCRIPTION
**Pull request by Reception123**

### CODE REVIEW

File: modules/mediawiki/manifests/init.pp [hunk 1]
```diff
@@ -97,6 +97,7 @@
     $recaptcha_secretkey  = lookup('passwords::recaptcha::secretkey')
     $googlemaps_key       = lookup('passwords::mediawiki::googlemapskey')
     $matomotoken          = lookup('passwords::mediawiki::matomotoken')
+    $googlemaps_key       = lookup('passwords::mediawiki::yandextranslationkey')
```
Comment (paladox) [inline modules/mediawiki/manifests/init.pp:100]: Should be yandextranslation_key

Comment (Reception123) [reply modules/mediawiki/manifests/init.pp:100]: Why is it not for google maps? https://github.com/miraheze/puppet/blob/9651400d4f605bfa730e1a445f36003855bcc135/modules/mediawiki/templates/PrivateSettings.php#L9

Comment (paladox) [reply modules/mediawiki/manifests/init.pp:100]: huh? There is already a key called 'googlemaps_key', this is a duplicate, not to mention the name is not correct for the what it'll be doing.

File: modules/mediawiki/templates/PrivateSettings.php [hunk 2]
```diff
@@ -25,6 +25,9 @@
 // Matomo Token
 $wgMatomoAnalyticsTokenAuth = "<%= @matomotoken %>";
 
+// Translate Yandex API key
+$wmgYandexTranslationKey = > "<%= @yandextranslation_key @>";
```
Comment (paladox) [inline modules/mediawiki/templates/PrivateSettings.php:29]: Should be ```$wmgYandexTranslationKey = "<%= @yandextranslation_key %>";```

File: modules/mediawiki/templates/PrivateSettings.php [hunk 3]
```diff
@@ -25,6 +25,9 @@
 // Matomo Token
 $wgMatomoAnalyticsTokenAuth = "<%= @matomotoken %>";
 
+// Translate Yandex API key
+$wmgYandexTranslationKey = "<%= @yandextranslation_key @>";
```
Comment (paladox) [inline modules/mediawiki/templates/PrivateSettings.php:29]: Should be ```$wmgYandexTranslationKey = "<%= @yandextranslation_key %>";```

**Pull request merged**